app/controllers/concerns/curation_concerns/application_controller_behavior.rb

```diff
@@ -34,6 +34,7 @@ def deny_access(exception)
         respond_to do |wants|
           wants.html do
             if [:show, :edit, :create, :update, :destroy].include? exception.action
+              flash[:alert] = exception.message
```

Comment:
I'm not sure we want to expose this as it's not an appropriate message for a non-programmer.